File: Lunchtime/Controller/SetupPageFourthViewController.m
//
//  SetupPageFourthViewController.m
//  Lunchtime
//
//  Created by Willow Belle on 2015-11-17.
//  Copyright © 2015 Cosmic Labs. All rights reserved.
//

#import "SetupPageFourthViewController.h"

@interface SetupPageFourthViewController ()

@property (nonatomic, weak) IBOutlet UIButton *shareNotificationsPrivilegeButton;

@end

@implementation SetupPageFourthViewController

- (IBAction)shareNotificationsPrivilegeButtonPressed:(UIButton *)sender {
    UIUserNotificationSettings *settings = [UIUserNotificationSettings settingsForTypes:UIUserNotificationTypeAlert categories:nil];
    [[UIApplication sharedApplication] registerUserNotificationSettings:settings];
}

@end

File: Lunchtime/Controller/SetupPageFourthViewController.swift
//
//  SetupPageFourthViewController.swift
//  Lunchtime
//
//  Created by Willow Bumby on 2015-11-18.
//  Copyright © 2015 Lighthouse Labs. All rights reserved.
//

import UIKit

class SetupPageFourthViewController: UIViewController {

    // MARK: - Outlets
    @IBOutlet weak var sharePrivilegeButton: UIButton?

    // MARK: - Actions
    @IBAction func sharePrivilegeButtonPressed(sender: UIButton) {
        let settings = UIUserNotificationSettings(forTypes: .Alert, categories: nil)
        UIApplication.sharedApplication().registerUserNotificationSettings(settings)
    }
}
